src/MICore/MICoreResources.resx

```diff
@@ -219,4 +219,7 @@ Error: {1}</value>
   <data name="Error_NoMiDebuggerPath" xml:space="preserve">
     <value>Unable to determine path to debugger.  Please specify the "MIDebuggerPath" option.</value>
   </data>
+  <data name="Error_InternalFileMissing" xml:space="preserve">
+    <value>An internal file could not be found.</value>
```

Comment:
I agree, but this message is being shown in VSCode. Are we still ok showing the file that couldn't be found there?